docs/changelog/89070.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 89070
+summary: "Fix: do not generate empty buckets for the date histogram"
+area: "Rollup"
+type: bug
```

Comment:
This PR should be labeled as `>test-failure` (not a `bug`) and it should contain no changelog file